charts/chart-colors.js

```diff
@@ -0,0 +1,321 @@
+export const ColorPalette = [
```

Comment:
Assuming JS files w/ a TS equivalent are generated, these should be part of a distributable and not committed.

This may also need an associated declaration (`d.ts`) file.